src/org/labkey/cds/CDSController.java

```diff
@@ -1370,4 +1373,30 @@ public void setReferrer(String referrer)
         }
 
     }
+
+    @RequiresPermission(ReadPermission.class)
+    public class GetReportsDataAction extends ReadOnlyApiAction<Object>
```

Comment:
GetReportsDataAction is used in both Study and Publication pages - see store/Study.js line 106 & store/Publication.js line 73
So makes sense to keep it "neutral" since its not tied to just Pub page.

Reply:
makes sense. thanks for clarifying